style.css

```diff
@@ -1,340 +1,1338 @@
-  height: 14px;
-  content: "";
-  border: 2px solid #333;
-  border-radius: 2px;
-  background-color: #fae6ed;
-}
-.folder_error .title span.title-btn span:before, .folder_error .title span.title-btn span:after {
-  content: "";
-  border-top: 2px solid #333;
-  background-color: #ddd;
-}
-.folder_error .title span.title-btn span.btn1:before {
-  position: absolute;
-  top: 11px;
-  left: 4px;
-  width: 10px;
-}
-.folder_error .title span.title-btn span.btn2:before {
-  position: absolute;
-  top: 4px;
-  left: 50%;
-  transform: translatex(-50%);
-  width: 6px;
-  height: 4px;
-  border: 2px solid #333;
-  border-top: 4px solid #333;
-}
-.folder_error .title span.title-btn span.btn3:before, .folder_error .title span.title-btn span.btn3:after {
-  position: absolute;
-  top: 8px;
-  right: 4px;
-  width: 10px;
-}
-.folder_error .title span.title-btn span.btn3:before {
-  transform: rotate(45deg);
-}
-.folder_error .title span.title-btn span.btn3:after {
-  transform: rotate(-45deg);
-}
-.folder_error div.body {
-  display: flex;
-  justify-content: space-around;
-  padding: 16px 32px;
-  font-size: 10px;
-  font-family: "Press Start 2P", cursive;
-  z-index: -30;
-}
-.folder_error div.body div.profile-icon-box {
-  position: relative;
-  margin: 0;
-  z-index: 0;
-}
-.folder_error div.body div.profile-icon-box p {
-  margin: 8px 0;
-  height: 14px;
-}
-.folder_error div.body div.profile-icon-box div.profile-icon {
-  position: relative;
-  display: inline-block;
-  margin: 8px 23px 20px 0;
-  border: solid 2px #111;
-}
-.folder_error div.body div.profile-icon-box div.profile-icon:before, .folder_error div.body div.profile-icon-box div.profile-icon:after {
-  position: absolute;
-  width: 120px;
-  height: 120px;
-  content: "";
-  border: solid 2px #111;
-  background: #fff;
-}
-.folder_error div.body div.profile-icon-box div.profile-icon:before {
-  top: 7px;
-  right: -13px;
-  z-index: -10;
-}
-.folder_error div.body div.profile-icon-box div.profile-icon:after {
-  top: 17px;
-  right: -25px;
-  z-index: -20;
-}
-.folder_error div.body div.profile-icon-box div.profile-icon img {
-  width: 120px;
-}
-.folder_error div.body div.name {
-  padding: 40px 0;
-  text-align: right;
-}
-.folder_error div.body div.name button.ok-error-btn {
-  display: inline-block;
-  margin-top: 8px;
-  padding: 6px 8px;
-  border: solid 1px #111;
-  border-right: solid 2px #111;
-  border-bottom: solid 2px #111;
-  background: pink;
-  color: white;
+p {
+    margin: 0 0 16px;
+}
+
+a {
+    color: var(--color-text);
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: none;
+    opacity: 0.6;
+}
+
+ul, ol {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+}
+
+h1, h2, h3, blockquote {
+    margin: 0;
+    padding: 0;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+    vertical-align: top;
+}
+
+button {
+    background: none;
+    border: 0;
+    outline: 0;
+    margin: 0;
+    padding: 0;
+}
+
+figure {
+    margin: 0 0 30px 0;
```

Comment:
Try to use shorthands.
No need to set last 0. 
Should be margin: 0 0 30px;

Reply:
ok) Done